app/models/cloud_volume.rb

```diff
@@ -18,6 +19,14 @@ class CloudVolume < ApplicationRecord
 
   acts_as_miq_taggable
 
+  supports :cloud_volume_backup_create do
+    unsupported_reason_add(:cloud_volume_backup_create, _("Backup unsupported")) unless validate_backup_create
```

Comment:
So it looks like it depends on the `raw_` methods being implemented.
I have no better idea than removing those here and doing `respond_to?` 
Or remove the `validate_backup_create`  and the `supports` block here and define `supports :cloud_volume_backup_create` on the subclass